doc/stdlib/libc.rst

```diff
@@ -40,7 +40,10 @@
 
 .. function:: time(t::TmStruct)
 
-   Converts a ``TmStruct`` struct to a number of seconds since the epoch.
```

Comment:
It's hard to follow this conversation purely over email, but what about
doing docstrings for multiple things by applying the docstring to a literal
array of objects? E.g.
>
>
"doc for `foo`, `bar` and `baz`"
[foo, bar, baz]


I can't see any reason one would ever need to actually attach documentation
to a literal array that isn't assigned anywhere, so this seems like it
would be a safe syntax to interpret specially. It might be nice to document
constant arrays on the other hand so that you can, e.g. write a doc string
for LOAD_PATH and then do things like

help?> LOAD_PATH


and get the docstring associated with that array object. Although perhaps
that should be done at the global symbol level instead of the object level.